### PR TITLE
fix bootstrap path

### PR DESCRIPTION
**Pull request by rajyan**
- [x] Added or updated tests
- [ ] Documented user facing changes
- [ ] Updated CHANGELOG.md

**Changes**
I want to re-open this PR https://github.com/nunomaduro/larastan/pull/851.
This PR would enable running larastan without caring about the working directory.
As I mentioned in https://github.com/nunomaduro/larastan/pull/851, Laravel's bootstrap and vendor path are hardcoded, and should always be a relative path `__DIR__.'/../../../bootstrap/app.php'` from `vendor/nunomaduro/larastan`.

I modified the test a little, and added a test to confirm the changes.

**Breaking changes**

I think there are no breaking changes.

### CODE REVIEW

File: bootstrap.php [hunk 1]
```diff
@@ -6,7 +6,7 @@
 
 define('LARAVEL_START', microtime(true));
 
-if (file_exists($applicationPath = getcwd().'/bootstrap/app.php')) { // Applications and Local Dev
+if (file_exists($applicationPath = __DIR__.'/../../../bootstrap/app.php')) { // Applications and Local Dev
```
Comment (szepeviktor) [inline bootstrap.php:9]:
```suggestion
if (file_exists($applicationPath = dirname(__DIR__, 3).'/bootstrap/app.php')) { // Applications and Local Dev
```

Comment (rajyan) [reply bootstrap.php]: Thank you!

**Review by szepeviktor — APPROVED** [review]

File: bootstrap.php (hunk 1 above)
Comment (szepeviktor) [reply bootstrap.php]: You're welcome.

**Comment by canvural** [comment]
Thank you!

**Comment by bastien-phi** [comment]
It looks like it breaks phpstan when using vendor-bin in an application as the `bootstrap/app.php` is not found in this case

I'm having 
```
PHP Fatal error:  Trait "Orchestra\Testbench\Concerns\CreatesApplication" not found in /code/my-project/vendor-bin/phpstan/vendor/nunomaduro/larastan/src/ApplicationResolver.php on line 16
```

**Comment by szepeviktor** [comment]
@bastien-phi Thank you for your report.
Could you be more specific on your setup? Maybe opening a brand new issue?

**Comment by canvural** [comment]
@bastien-phi Is this a Laravel app or a package?

**Comment by bastien-phi** [comment]
@szepeviktor @canvural see #856

**Pull request merged**